R/ml_kmeans.R

```diff
@@ -54,19 +56,33 @@ ml_kmeans <- function(x,
   # extract cluster centers
   kmmCenters <- invoke(fit, "clusterCenters")
 
+  # compute cost for k-means
+  if (compute.cost)
```

Comment:
How expensive is this computation? Is it something we could feasibly do by default?